src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -825,6 +825,16 @@ template <typename I>
 template <typename V>
 void WriteLog<I>::flush_pmem_buffer(V& ops)
 {
+  utime_t now = ceph_clock_now();
+  for (auto &operation : ops) {
+    if (operation->reserved_allocated()) {
+      operation->buf_persist_start_time = now;
```

Comment:
Ah, so `buf_persist_time` wasn't ever set before?

Reply:
> Ah, so `buf_persist_time` wasn't ever set before?

yes, this fix is the main purpose of this pr.